doc/user/content/manage/monitor/prometheus.md

```diff
@@ -0,0 +1,95 @@
+overall health of your Materialize instance using Prometheus and Grafana.
+
+## Before you begin
+
+Ensure you have:
+
+- A self-managed instance of Materialize installed with helm values `observability.enabled=true`, `observability.podMetrics.enabled=true`, and `prometheus.scrapeAnnotations.enabled=true`
+- [Helm](https://helm.sh/docs/intro/install/) version 3.2.0+ installed
+- [kubectl](https://kubernetes.io/docs/tasks/tools/) installed and configured
+
+{{< important >}}
+This guide assumes you have administrative access to your Kubernetes cluster and the necessary permissions to install Prometheus.
+{{< /important >}}
+
+## 1. Download our Prometheus scrape configurations (`prometheus.yml`)
+  Download the Prometheus scrape configurations that we'll use to configure Prometheus to collect metrics from Materialize:
+  ```bash
+  curl -o prometheus_scrape_configs.yml https://raw.githubusercontent.com/prometheus-community/helm-charts/refs/heads/main/charts/prometheus/values.yaml
+  ```
+
+
+## 2. Install Prometheus to your Kubernetes cluster
+
+  {{< note >}}
+  In this guide, we use the [prometheus-community](https://github.com/prometheus-community/helm-charts) Helm chart to install Prometheus.
+  {{< /note >}}
+
+
+1. Download the prometheus-community default chart values (`values.yaml`):
+   ```bash
+   curl -O https://raw.githubusercontent.com/prometheus-community/helm-charts/refs/heads/main/charts/prometheus/values.yaml
+   ```
+
+2. Within `values.yaml`, replace `serverFiles > prometheus.yml > scrape_configs` with our scrape configurations (`prometheus_scrape_configs.yml`).
+
+3. Install the operator with the updated `values.yaml`:
+   ```bash
+   kubectl create namespace prometheus
+   helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+   helm repo update
+   helm install --namespace prometheus prometheus prometheus-community/prometheus \
+   --values values.yaml
+   ```
+
+## 3. Optional. Visualize through Grafana
+
+1. Install the Grafana helm chart following [this guide](https://grafana.com/docs/grafana/latest/setup-grafana/installation/helm/).
+
+
+2.  Set up port forwarding to access the Grafana UI:
+    ```bash
+    MZ_POD_GRAFANA=$(kubectl get pods -n monitoring -l app.kubernetes.io/name=grafana -o custom-columns="NAME:.metadata.name" --no-headers)
+    kubectl port-forward pod/$MZ_POD_GRAFANA 3000:3000 -n monitoring
+    ```
+
+    {{< warning >}}
+  The port forwarding method is for testing purposes only. For production environments, configure an ingress controller to securely expose the Grafana UI.
+    {{< /warning >}}
+
+3. Open the Grafana UI on [http://localhost:3000](http://localhost:3000) in a browser.
+
+4. In the Grafana UI, add a Prometheus data source. In the Connection section, set the Prometheus server URL to `http://<prometheus server name>.<namespace>.svc.cluster.local:<port>`(e.g. `http://prometheus-server.prometheus.svc.cluster.local:80`).
```

Comment:
Add a Prometheus data source. In Grafana UI, under **Connections > Data sources**, 
- Click Add data source and select **promethueus**.
- In the Connection section, set th **Prometheus serverURL** to ...